### PR TITLE
fix some warnings in Tests + change minimal requirements

### PR DESCRIPTION
**Pull request by quentinql**

### CODE REVIEW

**Review by Zluurk — APPROVED** [review]

File: pypeman/remoteadmin.py [hunk 1]
```diff
@@ -327,7 +331,9 @@ def do_channels(self, arg):
         "List avaible channels"
         result = self.client.channels()
         print("\nChannel list:")
+        logger.warning(result)
```
Comment (feenes) [inline pypeman/remoteadmin.py]: why is this a warning?

File: pypeman/remoteadmin.py [hunk 2]
```diff
@@ -327,7 +331,9 @@ def do_channels(self, arg):
...
         for idx, channel in enumerate(result):
+            logger.warning(channel)
```
Comment (feenes) [inline pypeman/remoteadmin.py]: why is this a warning?

File: pypeman/tests/common.py [hunk 3]
```diff
@@ -76,7 +76,7 @@ def generate_msg(timestamp=None, message_content=default_message_content,
     return m
 
 
-class TestException(Exception):
+class ExceptionTest(Exception):
```
Comment (feenes) [inline pypeman/tests/common.py:79]: TstException?

**Review by feenes — APPROVED** [review]

**Review by jrmi — APPROVED** [review]
Like this one ;-)

**Pull request merged**